models/DataObject/ClassDefinition/Data/Href.php

```diff
@@ -731,4 +731,17 @@ public function unmarshal($value, $object = null, $params = [])
             return Element\Service::getElementById($type, $id);
         }
     }
+
+    /**
+     * @param $value1 Element\ElementInterface
+     * @param $value2 Element\ElementInterface
+     * @return bool
+     */
+    public static function isEqual($value1, $value2) {
```

Comment:
@weisswurstkanone please make that one not static and put a default on the abstract (`return false`)